depthai_ros_msgs/include/depthai_ros_msgs/plugin/DaiDatatypeRawImgFrameType.h

```diff
@@ -0,0 +1,40 @@
+enum class DaiDatatypeRawImgFrameType {
```

Comment:
Instead of redefining the enum, thoughts on #include the dai header and then `using A = B;`?